src/behaviours/draggable.js

```diff
@@ -75,6 +76,7 @@ export default function draggable(WrappedComponent) {
     }
 
     componentDidMount() {
+      PreventGhostClick(findDOMNode(this.node));
```

Comment:
I have a feeling this should live in `selectable`, since this class doesn't trigger the click/tap callback?

For example if we did:
`const EnhancedNode = selectable(node);`

Would we get the same double click event again?

Reply:
Good catch! Will move the call.